src/app/creator-profile-page/creator-profile-hodlers/creator-profile-hodlers.component.ts

```diff
@@ -28,7 +28,7 @@ export class CreatorProfileHodlersComponent {
   };
 
   getPage(page: number) {
-    if (this.lastPage != null && page > this.lastPage) {
+    if (this.lastPage !== null && page > this.lastPage) {
```

Comment:
is this a safe null check? I think maybe we should default to using the `isNil` helper from lodash for null comparisons just to be on the safe side and so there is no room for mistakes.